sdk/ios/RaygunNativeBridge.m

```diff
@@ -92,29 +94,37 @@ static uint64_t getMemorySize(void) {
     return value;
 }
 
-//==============================//
-
-
-
+// ============================================================================
+#pragma mark - BRIDGE FIELDS -
+// ============================================================================
 
 @implementation RaygunNativeBridge
 
 static CFTimeInterval startedAt; //Time that this object was created
 
-BOOL hasInitialized = FALSE;
+BOOL crashReportingInitialized = FALSE;
 
 //RUM events to capture and send to the react layer
 NSString *viewName = @"RCTView";
 NSString *onStart = @"ON_START";
 NSString *onPause = @"ON_PAUSE";
 NSString *onResume = @"ON_RESUME";
 NSString *onDestroy = @"ON_DESTROY";
+NSString *DEVICE_ID = @"iOS DEVICE";
 
-NSString *defaultsKey = @"__RAYGUN_CRASH_REPORTS__";
-
+// ============================================================================
+#pragma mark - INHERITED NATIVE MODULE STARTUP METHODS -
+// ============================================================================
 
 + (void)initialize {
     startedAt = processStartTime(); //Set the time that this bridge was initialised at
+
+    //Get the device id and store it
+    DEVICE_ID = [[[UIDevice currentDevice] identifierForVendor] UUIDString];
```

Comment:
The identifierForVendor method is not guaranteed on every iOS version. I would reuse the logic found here: 
https://github.com/MindscapeHQ/raygun4apple/blob/4a6886068dca525843b03f6bc2839ce3865de1bb/Sources/Raygun/RaygunUserInformation.m#L53

https://github.com/MindscapeHQ/raygun4apple/blob/4a6886068dca525843b03f6bc2839ce3865de1bb/Sources/Raygun/RaygunUserInformation.m#L72

Reply:
raygun4apple implementation has been applied